RELEASING.md

```diff
@@ -1,77 +1,93 @@
 
-1. Update the version in `package.json` to match that in `lib/version.ts`
+  > **Note**
+  > If the change is very small and doesn't warrant a changelog entry, run `yarn changeset --empty` and commit the resultant file in the `.changeset` directory.
 
-1. Stage the `CHANGELOG.md`, `lib/version.ts`, and `package.json` files
+- When the PR is merged into the `next` branch, the `next-release.yml` workflow uses the `changesets/action` to either create or update an existing PR that has the title `Version Packages for Release Candidates`.
 
-   ```shell
-   git add CHANGELOG.md lib/version.ts package.json
-   ```
+### :exclamation: To perform a release of release candidate packages
 
-1. Commit the files to the repo
+1. Checkout the `changeset-release/next` branch
 
    ```shell
-   git commit -m "Packaging for release vX.Y.Z-rcN"
+   git checkout changeset-release/next
    ```
 
-1. Tag the commit with an _annotated_ tag (required for the `--follow-tags` of the `push` command). Note that it's very important that the tag includes `-rc` as part of the tag name - this is how `shipit` will recognize that it's a release candidate, not a general release.
+1. Update the version string in the `lib/version.ts` file to match the version in the `package.json` file.
+
+1. While the branch is checked out, edit/remove any of the comments in the changed `CHANGELOG.md` files and commit them to the `changeset-release/next` branch.
+
+1. Once the files in the PR reflect the desired release changes, merge the `Version Packages for Release Candidates (rc)` PR into `next` - this triggers the release.
+
+1. The same `changesets/action` in the `next-release.yml` workflow will call `yarn release`, which builds and pushes the release candidates to `npmjs.org`.
+
+## Merging `next` into `main` (moving from pre-release to main release)
+
+When a major set of changes is about to be mass released from the `next` branch
+
+> **Warning**
+>
+> The next steps need to be confirmed
+
+1. Checkout the `next` branch
 
    ```shell
-   git tag -a vX.Y.Z-rcN -m "Release candidate N of version X.Y.Z"
+   git checkout next
    ```
 
-1. Push the files to the remote repository, along with the annotated tag.
+1. Take the `next` branch out of pre-release mode by running
 
    ```shell
-   git push origin shopify_api_next --follow-tags
+   yarn changeset pre exit
    ```
 
-1. Login to `shipit`, search for the _next_ environment for `shopify-api-js`, and press _Deploy_ on the appropriate commit (the commit description will be the version number).
+   And commit the changed files.
+
+1. Merge the `next` branch into `main`. This _should_ update the relevant `CHANGELOG.md` files on `main` with the changes from the release candidates.
```

Comment:
Is that to be done manually ? `Changeset` does not do it automatically ?

Reply:
`changeset` does not do what automatically?